File: WEEK-2/week2a.c
#include<stdio.h>
void main(){
    int n;
    printf("enter an integer:\n");
    scanf("%d",&n);
    int num = 1;
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j <= i; j++)
        {
            printf("%d\t",num);
            ++num;
        }
        printf("\n");
    }
}

File: WEEK-2-IN-JAVA/week2a.java
import java.util.Scanner;

// import jdk.javadoc.internal.doclets.formats.html.SourceToHTMLConverter;

class week2a {
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number-");
        int n = sc.nextInt();
         
            int num = 1;
            for (int i = 0; i < n; i++){
                for (int j = 0; j <= i; j++){
                    System.out.print(num + " ");
                    ++num;
                }
                System.out.println();

            }
        
    }
}